examples/seismic/abc_methods/01_introduction.ipynb

```diff
@@ -0,0 +1,1192 @@
+    "- $x_{F}$ =  1.0 Km = 1000 m;\n",
+    "- $z_{I}$ =  0.0 Km;\n",
+    "- $z_{F}$ =  1.0 Km = 1000 m;\n",
+    "\n",
+    "with\n",
+    "\n",
+    "- $\\Delta x$ = 0.01 km = 10m;\n",
+    "- $\\Delta z$ = 0.01 km = 10m;\n",
+    "\n",
+    " The integration will be carried out from \n",
+    "\n",
+    "- $t_{I}$ = 0 s = 0 ms;\n",
+    "\n",
+    "to\n",
+    "\n",
+    "- $t_{F}$ = 1 s = 1000 ms;\n",
+    "\n",
+    "The choice of $\\Delta t$ has to respect the CFL condition:\n",
+    "\n",
+    "\\begin{equation}\n",
+    "{\\Delta t}\\leq\\displaystyle\\frac{h\\nu}{v_{max}},\n",
+    "\\end{equation}\n",
+    "\n",
+    "where $\\nu\\leq\\frac{1}{\\sqrt{2}}$ and\n",
+    "\n",
+    "\\begin{equation}\n",
+    "h = \\min\\{ \\Delta x, \\Delta z \\}.\n",
+    "\\end{equation}\n",
+    "\n",
+    "The term $v_{max}$ is the maximum velocity propagation in $\\Omega$ and is given by:\n",
+    "\n",
+    "\\begin{equation}\n",
+    "v_{max}=\\max_{(x,z)\\in \\Omega}\\left\\vert c(x,z) \\right\\vert.\n",
+    "\\end{equation}\n",
+    "\n",
+    "The choice of $\\Delta t$ respecting the CFL condition defines the number of timesteps $NT$:\n",
+    "\n",
+    "\\begin{equation}\n",
+    "\\Delta t =\\frac{T}{NT}.\n",
+    "\\end{equation}\n",
+    "\n",
+    "In all the simulations we employ $\\nu=0.4 $. In this way, the temporal parameters are given by:\n",
+    "\n",
+    "- $\\Delta t$ $\\approx$ 0.0016 s = 1.6 ms;\n",
+    "- $NT$ = 626.\n",
+    "\n",
+    " The source $f(x, z, t)$  will be a Ricker source with the following properties:\n",
+    "\n",
+    "- Postion at $x:$ $\\bar{x} = 500 m = 0.5 Km$;\n",
+    "- Position at $z:$ $\\bar{z} = 10 m = 0.01 Km$;\n",
+    "- Peak frequence: $f_{0} = 10 Hz = 0.01 Khz$;\n",
+    "\n",
+    "The graph of $f(\\bar{x}, \\bar{z}, t)$ will be generated when building the code. We employ a synthetic velocity model $c(x,z)$ with the following properties:\n",
+    "\n",
+    "- Minimum propagation speed: $v_{min} = 1500 m/s = 1,5 Km/s$;\n",
+    "- Maximum propagation speed: $v_{max} = 2500 m/s = 2,5 Km/s$;\n",
+    "\n",
+    "The figure of the velocity profile will be generated when building the code. We will consider 'receivers' along the $x$ direction (at all discrete points between $0.0$ Km and $1.0$ Km) at the depth $z=0.01$ Km, where we measure the displacement $u(x,z,t)$ throughout the time integration. Based on these values at the receivers we bild a $seismogram$ at the end of the integration."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 1.5 - Numerical Simulations\n",
+    "\n",
+    "In the numerical simulations, we initially import the packages of interest, in particular, those that are part of the Devito library. First we import the basic Python packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy                   as np\n",
```

Comment:
@FabioLuporini @mloubout This style is not consistent with our other tutorials. Do we want it made consistent or fine as is?

Reply:
If there is any standardization in the call of libraries, we can put these calls in a more appropriate format.